Gemfile

```diff
@@ -32,7 +32,8 @@ gem 'jsbundling-rails' # Bundle and transpile JavaScript
 gem 'i18n-js', '3.9.0' # A library to provide the I18n translations on the Javascript
 # gem 'devise-i18n' # Translations for Devise
 
-gem 'psych', '< 4.0'
+# Fix Ruby 3.1 that uses psych version 4.0 and conflicts with rails-template
+gem 'psych', '3.3.4'
```

Comment:
the root cause is that Ruby 3.1 comes with psych v4 which is a breaking change. But the gem file here is limiting the version to `< 4` already. So why we need to set a specific version `3.3.4`?

Reply:
The line gem 'psych', '< 4.0, is that I accidentally committed it with initial commit. I think it can make some confused, so I set it to a specific version to have a clear understanding, and 3.3.4 is the latest version before 4.0.